client/src/components/NationwideMap.vue

```diff
@@ -70,6 +77,15 @@ export default defineComponent({
     },
   },
   methods: {
+    zoomToLongLat() {
+      if (this.geoState?.geoids?.lat != null && this.geoState?.geoids.long != null) {
+        const longLat: LngLatLike = {
+          lon: parseInt(this.geoState?.geoids?.long),
```

Comment:
super nit: "long" so it's consistent

Reply:
It actually has to be lon for mapbox type inference, but I changed other references to lon for consistency.